extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -71,6 +71,28 @@
         </config-description>
     </thing-type>
 
+    <thing-type id="0200">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="gateway"/>
+        </supported-bridge-type-refs>
+
+        <label>Color Light</label>
+        <description>A dimmable light that supports full colors and color temperature settings.</description>
+
+        <channels>
+            <channel id="brightness" typeId="brightness"/>
```

Comment:
The color channel already accepts all the same (PercentType) commands as the brightness channel, so a thing that has the color channel, should not have the brightness channel as well as this is redundant.
See e.g. also the hue + LIFX bindings where it is the same.

Reply:
Ok, I can remove the channel. But I think that we loose functionality:
Imagine a group gWohnzimmer_Brightness for the living room.
In this group we have
* Something_Wohnzimmer_Brightness (something something)
* Tradfri_Wohnzimmer1_Brightness (dimmable bulb)
* Tradfri_Wohnzimmer2_Brightness (dimmable color temperature bulb)
* Tradfri_Wohnzimmer3_Brightness (full color bulb)

Further we have a `Slider item=gWohnzimmer_Brightness` in the sitemap.
Slide the slider and see how all lamps, also the full color bulb, change their brightness.
Without the brightness channel, one would have to use some rules, I think?
Or do I have a knot in my brain?

Reply:
Iirc when sending only the brightness value (0 - 100) to a color item it will be interpreted as brightness only. So in your example all bulbbs should change their brightness.

Reply:
Ah well I think you're right.
Will verify this and update my PR accordingly.

Reply:
You remember correctly! The [ColorItem](https://github.com/eclipse/smarthome/blob/74a486dfbf8201a2c48f021fa2ef4353a5ba2ca8/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/ColorItem.java#L83) takes care of this: If it gets updated with a percent type, it keeps everything else from the previous state and only adjusts the brightness.

Reply:
I tried this, but got unexpected results.
My steps:
* Breakpoints on `TradfriLightHandler.handleCommand` and `ColorItem.setState`
* Item: tradfri_0200_gwa0cc2b6d64bb_65551_color
* Set a color with BasicUI -> handleCommand receives HSBType -> good
* Send "100" to the same item -> handleCommand receives PercentType -> unexpected?
* AFTER this, the breakpoint on `ColorItem.setState` kicks in
My current code has already a special handling for PercentType on the color channel, so the functionality is OK. But the ColorItem.setState doesn't help here, I think.
What's your opinion about this?